public/sandbox-data/apprenticeship-levy/employers/222hmrctest3/employed/AA345678C.json

```diff
@@ -0,0 +1,10 @@
+{
+  "status": 200,
+  "jsonBody": {
+    "empref": "222/hmrctest3",
+    "nino": "AA345678C",
```

Comment:
Use of Nino

Reply:
Ill address this in a later ticket. This is a bigger piece of work as App-levy is using itself as a stub. We need to create a separate microservice for this.